### PR TITLE
add dmail dapp

### PR DESCRIPTION
**Pull request by bin-dogami**

### CODE REVIEW

File: public/dapps/dmail/dmail-banner.png
Comment (graemeblackwood) [inline public/dapps/dmail/dmail-banner.png:1]: @bin-dogami The text will br cropped on smaller screen sizes. I recommend keeping anything important within the safe area defined in this Figma file: https://www.figma.com/community/file/1163928128813560560/dappland-content-template

**Comment by graemeblackwood** [comment]
@bin-dogami Thanks for this. It looks like you're using an older version of the data JSON format. Updating `dotw` to `dotm` might fix it.

Also you list one contract, which is Verified (great to see!). Is this the only contract used by DMail?

**Comment by bin-dogami** [comment]
> @bin-dogami Thanks for this. It looks like you're using an older version of the data JSON format. Updating `dotw` to `dotm` might fix it.
> 
> Also you list one contract, which is Verified (great to see!). Is this the only contract used by DMail?

hi,  I replaced the dotw` with `dotm`, is it right?
Also, I removed the contract, we have no contract yet.

**Comment by graemeblackwood** [comment]
Thanks @bin-dogami – It looks like the app is live despite having no contracts. I am struggling to test it though because it wants a "Principal ID" for my recipient address (my other wallet).

**Pull request merged**